### PR TITLE
:bug: ApplicationForm: Force onChange when unBlur Owner field

### PR DESCRIPTION
**Pull request by gildub**
Resolves https://issues.redhat.com/browse/MTA-651

### CODE REVIEW

File: client/src/app/pages/applications/components/application-form/application-form.tsx [hunk 1]
```diff
@@ -505,7 +505,7 @@ export const ApplicationForm: React.FC<ApplicationFormProps> = ({
             name="owner"
             label={t("terms.owner")}
             fieldId="owner"
-            renderInput={({ field: { value, name, onChange } }) => (
+            renderInput={({ field: { value, name, onChange, onBlur } }) => (
```
Comment (ibolton336) [inline client/src/app/pages/applications/components/application-form/application-form.tsx:508]: Is this ever used?

Comment (gildub) [reply client/src/app/pages/applications/components/application-form/application-form.tsx:508]: @ibolton336 , yes it's used below, line 528.
That's the workaround I found to force the field to be empty in relation of the associated MTA-651 bug.

Comment (ibolton336) [reply client/src/app/pages/applications/components/application-form/application-form.tsx:508]: Looks like you're never passing the onBlur. Rather, you are passing onChange as the argument to the onBlur prop for simpleSelect.

File: client/src/app/pages/applications/components/application-form/application-form.tsx [hunk 2]
```diff
@@ -525,6 +525,7 @@ export const ApplicationForm: React.FC<ApplicationFormProps> = ({
                   const selectionValue = selection as OptionWithValue<string>;
                   onChange(selectionValue.value);
                 }}
+                onBlur={onChange}
```
Comment (ibolton336) [inline client/src/app/pages/applications/components/application-form/application-form.tsx:528]: Did you mean onBlur here?

Comment (gildub) [reply client/src/app/pages/applications/components/application-form/application-form.tsx:528]: Yes. That works around the issue but maybe there is better way.

File: client/src/app/pages/applications/components/application-form/application-form.tsx (hunk 1 above)
Comment (gildub) [reply client/src/app/pages/applications/components/application-form/application-form.tsx]: @ibolton336, gotcha, I thought I was using that onBlur ! I removed it.

**Review by ibolton336 — APPROVED** [review]

**Comment by ibolton336** [comment]
Looks like your force push here has regressed the url change. @gildub

**Pull request merged**